utils/oauth.go

```diff
@@ -0,0 +1,199 @@
+package utils
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"userstyles.world/config"
+)
+
+var (
+	githubClientID   = "f3f2d378d88794062895"
+	gitlabClientID   = "f906330603e645d54184fd13337ca51b762ea5da2188e7f248e109c686940897"
+	codebergClientID = "308920d9-e49d-4db6-baa9-7513c9f6dea2"
```

Comment:
Should this also be turned into environment variables?

Reply:
It can, but they aren't a secret and just hardcoded 🤷🏽‍♀️

Reply:
I was thinking of using different client IDs in development and production environments.